### ISSUE TITLE
Preferences reorganization and setup wizard

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#97]
```
Goals:
- show only a minimum of preferences at first (e.g. all that is needed to get 
GTalkSMS going)
- move all not so common settings to an "advanced" tab
- write an setup wizards, the guides the user through the configuration
- recheck if all pre-defined settings are reasonable and in the right category

Further suggestions welcome.

```

Original issue reported on code.google.com by `fschm...@gmail.com` on 11 Feb 2011 at 8:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 13 Feb 2011 at 4:27
- Changed title: **Preferences reorganization and setup wizard**

**Comment by GoogleCodeExporter** [comment]
```
Introduced "Advanced Settings" for setting categories
```

Original comment by `fschm...@gmail.com` on 11 Mar 2011 at 4:27
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 14 Jun 2011 at 7:23
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 11 Jul 2011 at 9:22
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v3.0
```

Original comment by `Florent....@gmail.com` on 15 Jul 2011 at 3:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
